txpipe/data_types/types.py

```diff
@@ -27,6 +31,23 @@ def read_catalog_info(self):
         shear_catalog_type = info.get('catalog_type')
         return shear_catalog_type
 
+    @property
```

Comment:
just curious, what does this do?

Reply:
@property is a general python decorator that turns a class method that you have to call into an attribute that you can just get with a dot, so in this case you can just do  `t = shear_cat.catalog_type` not e.g. `t = shear_cat.get_catalog_type()`.   Here's a nice description: https://www.programiz.com/python-programming/property